tests/utils.py

```diff
@@ -20,7 +20,10 @@ def zip_strict(left: Sequence[Any], right: Sequence[Any]) -> Iterator[Any]:
 def compare_dicts(result: Any, expected: dict[str, Any]) -> None:
     if hasattr(result, "collect"):
         result = result.collect()
-    if hasattr(result, "columns"):
+    if hasattr(result, "column_names"):  # To handle Pyarrow ChuckedArray:

```

Comment:
`result.columns` returned a `ChuckedArray` for this one which was throwing me an error.

Reply:
I think the idea is to compare with narwhals dataframe directly, not converting back to native :)